index.html

```diff
@@ -213,6 +219,7 @@ <h2 class="h2">
         
     </section>
```

Comment:
- [ ] Kindly the section1 don't have a background as you can see in the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m4_desktop/desktop.png) template. please add the background for this section 

Your Design
![Screenshot from 2022-05-07 14-35-26](https://user-images.githubusercontent.com/77842237/167252990-65d39b24-39ea-43a6-b1cb-2d8bdb84d380.png)

Reply:
@vic778 it have background in my browser I don't now why it's not displayed with you.

Reply:
[photo](C:\Users\pc\Pictures\Screenshots)

Reply:
### OPTIONAL

Hi @Abel-desu when I clone your repository I see the background image, please try to make it look more like the Figma design currently, your image looks aligned to the left on 768 and 1024 px, please adjust it, please note that this change is optional👍🏿

![Screenshot_158](https://user-images.githubusercontent.com/69638013/167257075-3e2ab471-4e51-4e90-9fa4-64b6c5be87b8.png)